Forwarded: https://github.com/maoschanz/drawing/commit/c63b496232540dbce5da1b5e872c1a3968bb3825
Author: "Romain F. T" <rrroschan@gmail.com>
Last-Update: 2022-10-12
Description: [PATCH] wrong default option for text background (#521)
Ubuntu patch for 0.8.5 removes the new v1.0 shapes

---
 data/com.github.maoschanz.drawing.gschema.xml |  2 +-
 src/tools/classic_tools/tool_shape.py         | 52 +++++++++++--------
 2 files changed, 30 insertions(+), 24 deletions(-)

diff --git a/data/com.github.maoschanz.drawing.gschema.xml b/data/com.github.maoschanz.drawing.gschema.xml
index 70eac4ff..d662c412 100644
--- a/data/com.github.maoschanz.drawing.gschema.xml
+++ b/data/com.github.maoschanz.drawing.gschema.xml
@@ -171,7 +171,7 @@
       tool, saved when closing a window.</description>
     </key>
     <key type="s" name="last-text-background">
-      <default>'outline'</default>
+      <default>'thin-outline'</default>
       <summary>Text insertion background style</summary>
       <description>The name of the last active background style, from the "text"
       tool, saved when closing a window.</description>
-- 
2.34.1

